packages/nx/src/tasks-runner/utils.ts

```diff
@@ -182,15 +182,19 @@ export function getOutputsForTargetAndConfiguration(
   }
 }
 
-export function interpolate(template: string, data: any): string {
+export function interpolate(
+  template: string,
+  data: any,
+  replaceAsUndefinedIfNotFound: boolean = true
```

Comment:
I don't think we need to add this flag. If you remove it we can merge it.

Reply:
I've chosen to do it like this to keep it backward compatible for the other locations where this exported function is used (my first local change was without it and I saw it broke a test for which I could not judge on the consequences at first sight).
It's now bed time for me, but I'll have a further look at it tomorrow if that is your preference.
Thanks already!

Reply:
Couldn't hold myself; there was only 1 other usage of the method and one broken test, so in the end not that hard to fix.